flow/core/kernel/scenario/traci.py

```diff
@@ -0,0 +1,885 @@
+"""Script containing the TraCI scenario kernel class."""
```

Comment:
I'm currently assuming this is copied over from other code unchanged, if it is not and needs review, let me know

Reply:
Nope completely copied! Don't worry about it